src/client/App.js

```diff
@@ -11,20 +11,33 @@ import Profile from './containers/Profile';
 import Loader from './components/Loader';
 import NotFoundPage from './containers/NotFound/notFoundPage.component';
 import Footer from './components/Footer/Footer.js';
+import { CVreviews } from './components/CVreviewsPage/CVreviews';
+import { AddResume } from './components/AddResume/AddResume';
 
 function App() {
+  const [uploadedFile, setUploadedFile] = useState('');
   const { isAuthenticated, isLoading } = useAuthentication();
   if (isLoading) return <Loader />;
   return (
     <Router>
       {location.pathname === '/sign-in' ||
       location.pathname === '/sign-up' ? null : (
-        <Header isAuthenticated={isAuthenticated} username="William Henry Gates" />
+        <Header
+          isAuthenticated={isAuthenticated}
+          username="William Henry Gates"
+        />
       )}
       <Switch>
         <Route exact path="/">
           <Home />
         </Route>
+        <AddResume
+          uploadedFile={uploadedFile}
+          setUploadedFile={setUploadedFile}
+          exact
+          path="/add-resume"
+        />
```

Comment:
@hamedveloper  There might be better to avoid ```Prop drilling``` by the help of  a context, then context providerder will send ```uploadedFile``` and  ```setUploadedFile``` easily to  many components such as ```CVreviews``` , ```Addresume```  and later on to  ```Profile```

Reply:
Something like these:
```
    <yourHandlerContext.Provider value={{ uploadedFile, setUploadedFile }}>
        <AddResume    
          exact
          path="/add-resume"
        />
        <CVreviews exact path="/cv-reviews" />
      </yourHandlerUserContext.Provider>
```

Reply:
@hamedveloper  I think ```<AddResume>```  and ``` <CVreviews>``` need to be placed between ```AuthenticatedRoute``` because they just accessible by the authorized users.  check for this I line 44